index.js

```diff
@@ -96,7 +98,27 @@ module.exports = async params => {
 	}
 
 	const {packageJson} = readPkgUp.sync();
-	const {appId} = packageJson.build;
+	const buildConfig = packageJson.build;
+	let appId = '';
+
+	if (buildConfig) {
+		// appId from ./package.json
+		appId = buildConfig.appId;
+	} else {
+		try {
+			// fallback for vue-cli-plugin-electron-builder
+			// it uses vue.config.js to create electron builder config yaml file
+			// https://github.com/nklayman/vue-cli-plugin-electron-builder
+			const path = './dist_electron/builder-effective-config.yaml'
+			if (fs.existsSync(path)) {
```

Comment:
Remove this check. I believe this file is always present in an electron build, so throwing if it's not should be ok.